src/networkextension.cs

```diff
@@ -1293,7 +1293,11 @@ interface NEFilterSocketFlow {
 				
 	[iOS (9,0)][Mac (10,11, onlyOn64 : true)]
 	[BaseType (typeof (NETunnelNetworkSettings))]
+	[DisableDefaultCtor]
```

Comment:
Ok, I kept the default constructor, although it's useless since the resulting NETunnelNetworkSettings object can't be used to initiate the tunneling interface.